odxtools/diaglayers/diaglayertype.py

```diff
@@ -1,5 +1,8 @@
 # SPDX-License-Identifier: MIT
 from enum import Enum
+from typing import NewType
+
+TInheritancePrio = NewType("TInheritancePrio", float)
```

Comment:
* why the T prefix ?
* int or float ?

Reply:
T for Type 😄
The float is intentional, see my other comment.

Reply:
T prefix is generally used with generic types, I don't see it being one here.

Example

```
from typing import TypeVar, List

TItem = TypeVar('TItem')  # TItem can be any type

def first_element(elements: List[TItem]) -> TItem:
    return elements[0] 
```

Reply:
I renamed it to `InheritancePriority`